src/appengine/libs/issue_management/oss_fuzz_github.py

```diff
@@ -212,3 +218,20 @@ def close_issue(testcase):
   _close_issue_with_comment(testcase, issue)
   logs.log(f'Closed issue number {testcase.github_issue_num} '
            f'in GitHub repository {testcase.github_repo_id}.')
+
+
+def get_my_issues():
+  """Get all issues filed by oss-fuzz-robot."""
+  access_token = _get_access_token()
+  return access_token.search_issues('author:oss-fuzz-robot')
+
+
+def get_bug_information(issue):
```

Comment:
This is pretty specific to the butler script and unlikely to be used elsewhere. Let's move it there.